spec/factories/sales_log.rb

```diff
@@ -128,8 +128,7 @@
       pregla { 1 }
       pregother { 1 }
       pregghb { 1 }
-      hhregres { 1 }
-      hhregresstill { 4 }
+      hhregres { 7 }
```

Comment:
This made some debugging easier, and exposed a few places that shouldn't have been relying on the old values so I fixed those and left the change in.